doc/source/math/direction.rst

```diff
@@ -185,6 +185,13 @@ Structure
 
     :struct:`Vector` of length 1 that is in the same direction as the "starboard side" of this Direction.  Note that it is the same meaning as "what the X axis of the universe would be rotated to if this rotation was applied to the basis axes of the universe". When you LOCK STEERING to a direction, that direction's STARVECTOR is the vector the right wing of the ship will orient to.  SHIP:FACING:STARVECTOR is the way the ship's right wing is aimed right now.
 
+.. attribute:: Direction:INVERSE
+
+    :type: :struct:`Direction`
+    :access: Get only
+    
+    :struct: Gives a `Direction` with the opposite rotation around its axes. So `r(30,15,5)` becomes `r(-30,-15,-5)`.
+    
```

Comment:
Is this true?  I was unsure.  (remembering of course, that if it says, say, 330 that does mean -30 - I think Rots get printed clamped into the 0-360 range so you never see negative numbers when you print them (i.e. -10 prints as +350).)

but I don't remember if this is really true - has this claim been tested?